lib/python/qmk/cli/new/keymap.py

```diff
@@ -0,0 +1,65 @@
+"""
+This script automates the copying of the default keymap into your own keymap.
+"""
+import os
+import sys
+
+from milc import cli
+from qmk.path import keymap, check_directories, create_keymap_directory
+
+
+# TODO(orngshdw): add debugging code if needed:
+#   @cli.argument('--debug', dest='debug', action='store_true', help='specify to turn on debugging messages')
+@cli.entrypoint('Create a new keymap.c from the default keyboard of your choosing')
+def main(cli):
+
+    # get keyboard
```

Comment:
I'd like it if the user can provide these options as command line arguments, with a fallback to asking the user for any information not provided. This is not strictly necessary to merge this PR if you'd like to leave it to a future PR (by yourself or someone else.)